processor/k8sprocessor/kube/kube.go

```diff
@@ -39,10 +39,6 @@ type PodIdentifier string
 
 var (
 	// TODO: move these to config with default values
```

Comment:
We keep this because the other two vars will need to be updated in the future, too, right?

Reply:
yep!